src/commands/sftp-user/create.tsx

```diff
@@ -0,0 +1,133 @@
+import { ExecRenderBaseCommand } from "../../lib/basecommands/ExecRenderBaseCommand.js";
+import {
+  makeProcessRenderer,
+  processFlags,
+} from "../../rendering/process/process_flags.js";
+import { ReactNode } from "react";
+import { Flags } from "@oclif/core";
+import { assertStatus } from "@mittwald/api-client-commons";
+import { Success } from "../../rendering/react/components/Success.js";
+import { Value } from "../../rendering/react/components/Value.js";
+
+import { projectFlags } from "../../lib/resources/project/flags.js";
+import { MittwaldAPIV2 } from "@mittwald/api-client";
+
+type Result = {
+  sftpUserId: string;
+};
+
+export class Create extends ExecRenderBaseCommand<typeof Create, Result> {
+  static summary = "Create a new sftp user";
+  static flags = {
+    ...projectFlags,
+    ...processFlags,
+    description: Flags.string({
+      required: true,
+      summary: "Description of sftp user",
+    }),
+    "public-key": Flags.string({
+      summary: "Public Key used for authentication",
+      exactlyOne: ["public-key", "password"],
+    }),
+    password: Flags.string({
+      summary: "Password used for authentication",
+      exactlyOne: ["public-key", "password"],
+    }),
+    expires: Flags.string({
```

Comment:
Please use the predefined flags for handling expiration dates (see `src/lib/flags/expireFlags.ts`).